tests/test_pro/test_pro_turbulence.py

```diff
@@ -6,7 +6,7 @@
 
 @pytest.mark.solve
 @pytest.mark.fluent_231
-def test_pro_exp(launch_fluent_solver_3ddp_t2):
+def test_pro_turbulence(launch_fluent_solver_3ddp_t2):
     if not os.path.exists("out"):
```

Comment:
Hi @prmukherj these files are already updated e.g. 
if not os.path.exists("out"):
        os.mkdir("out")  
in my another PR (Merged). Could you pls fetch the latest changes?

Reply:
@mkundu1, I think we have to take care of the merge conflicts in the pro-tests.

Reply:
@mkundu1, @akulshre-qa , @seanpearsonuk , FYI, the merge conflicts have been resolved.
Thank you